src/collections/_documentation/workflow/alerts-notifications/notifications.md

```diff
@@ -0,0 +1,91 @@
+        
+        1. When an individual resolves it by manually changing the state on the issue page UI
+        2. Submitting a fix with the associated issue ID
+        3. When the project’s auto-resolve feature is configured
+
+    - Auto-resolve is located in **Project Settings > General Settings**. For more details about submitting a fix, see documentation on [resolving issues](https://docs.sentry.io/workflow/releases/#after-associating-commits).
+    
+        [{% asset notifications/auto-resolve.png alt="A slider to indicate whether the auto resolve is disabled or not." %}]({% asset notifications/auto-resolve.png @path %})
+
+3. Ignored
+
+    - Ignored issues suppress alerts for the issue, and by default, are hidden from your project’s issue stream. Search the issue stream for `is:ignored` to view them.
+    
+    - Keep in mind; **an ignored issue will still count towards your quota.**
+
+Workflow emails are sent for issue state changes.
+
+## Deploy
+
+Deploy emails are sent to users who have committed to the release that was deployed. See [LINK: deploy documentation] for more details
+
+[ screenshot deploy-emails.png ]
+
+## Quota
+
+Sentry sends all owners of the organization emails when
+
+- The event volume is approaching or has exceeded the quota
+- On-demand is activated
+- Spike protection is activated or deactivated
+
+## **Notification Subscription**
+
+You can configure your personal workflow and deploy notifications [here](https://sentry.io/settings/account/notifications/). You **cannot configure** quota notifications.
+
+### **Workflow Notification Settings**
+
+[{% asset notifications/workflow_notifications_subscription.png alt="Radio buttons with options for always, ony on issues I subscribe to, and never." %}]({% asset notifications/workflow_notifications_subscription.png @path %})
+
+After selecting the appropriate alert setting, selectively change it by project in **User Settings > Account > Fine tune alerts by project**.
+
+Each project has three options: Default, On, or Off. Selecting default uses your default preference from the previous step.
+
+[{% asset notifications/fine_tune_workflow.png alt="A dropdown with the options to choose default, always, only on issues I subscribe to, and never." %}]({% asset notifications/fine_tune_workflow.png @path %})
+
+**Unsubscribe**
+
+To opt-out of workflow notifications for a specific issue, click "Unsubscribe" at the bottom right of the issue’s page.
+
+[{% asset notifications/unsubscribe.png alt="An unsubscribe button to stop notifications." %}]({% asset notifications/unsubscribe.png @path %})
+
+### Deploy Notifications
+
+Deploy notification settings are located in **Setting > Account > Notifications**.
+
+[{% asset notifications/deploy_notifications.png alt="An unsubscribe button to stop notifications." %}]({% asset notifications/deploy_notifications.png @path %})
+
+## **FAQ**
+
+- Does assigning a resolved issue trigger a notification email?
```

Comment:
Only one FAQ?  I assume we have some things we can point people to about noise reduction somewhere :)

Reply:
Yeah, these FAQs came from common Support questions. @adhiraj do you have suggestions for more FAQs?

Reply:
these are FAQs only for workflow notifications, alert FAQs are in a separate section. re: noise, I think a big factor is people just don't know the difference between these diff. notification types, and how to stop them. after we've elaborated all this in the docs, that should be clear.